frontend/src/app/Providers.tsx

```diff
@@ -1,14 +1,24 @@
 "use client";
+
 import React from "react";
 import { AuthProvider } from "@/contexts/AuthContext";
-import MessageReceiver from "@/app/MessageReceiver";
 import { ThemeProvider } from "next-themes";
+import { Toaster } from "@/components/ui/toaster";
+import { ToasterProps } from "sonner";
 
 const Providers = ({ children }: { children: React.ReactNode }) => {
+  const toastsConfig: ToasterProps = {
```

Comment:
Why not move this logic to a dedicated wrapper for the Toaster? Since it’s only related to toast notifications, this area should be reserved for more generic code related to all (or at least two) providers. Consider adding a `<ToasterWrapper />` component, where you can define the `toastsConfig` and then return `<Toaster {...toastsConfig} />`.

Reply:
Good call